docs/models/index.md

```diff
@@ -1,16 +1,67 @@
-so we've implemented key features related to learning events validation and
-conversion. For now, we mostly focus on two learning events standards: [Open
-edX](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/analytics.html)
-and [xAPI](https://adlnet.gov/projects/xapi/).
-
-Data validation and serialisation/de-serialisation are achieved using
-[pydantics](https://pydantic-docs.helpmanual.io) models that are documented in
-the following subsections:
-
-- [Open edX events](./edx/)
-- [xAPI events](./xapi/)
+# Learning statement models
+
+The learning statement models validation and conversion tools in Ralph empower you to work with an LRS and ensure the quality of xAPI statements.
+This features not only enhance the integrity of your learning data but also facilitates integration and compliance with industry standards.
+
+This section provides insights into the supported models, their conversion, and validation.
+
+## Supported statements
+
+Learning statement models encompass a wide array of xAPI and OpenEdx statement types, ensuring comprehensive support for your e-learning data.
+
+1. **xAPI statements models**:
+    - [LMS](https://profiles.adlnet.gov/profile/c4e8732c-428a-4aa0-9585-91bebcdea91d)
+    - [Video](https://profiles.adlnet.gov/profile/90b2c849-d744-4d0c-8bd0-403e7859a35b)
+    - [Virtual classroom](https://profiles.adlnet.gov/profile/4719f43e-28ef-4108-b76a-5fbde91c6f68)
+
+2. **OpenEdx statements models**:
+    - [Enrollment](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#enrollment-events)
+    - [Navigational](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#course-navigation-events)
+    - [Open Reponse Assessment](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#open-response-assessment-events)
+    - [Peer instruction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#peer-instruction-events)
+    - [Problem interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#problem-interaction-events)
+    - [Textbook interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#textbook-interaction-events)
+    - [Video interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#video-interaction-events)
+
+## Statements validation
+
+In learning analytics, the validation of statements takes on significant importance. 
+These statements, originating from diverse sources, systems or applications, must align with specific standards such as [xAPI](https://xapi.com/) or [Caliper](https://www.imsglobal.org/activity/caliper), for the best known. 
```

Comment:
Do we support Caliper ? If not, I would leave it out to avoid confusing the reader.